modules/envoy-gateway/create-synapse-oauth-client.py

```diff
@@ -0,0 +1,26 @@
+import synapseclient
```

Comment:
I'm wondering how this file is triggered or used?

Reply:
It's not at the moment. I originally had the thought that we might be able to use Synapse for the Oauth client credential flow (I learned we cannot), however, we will eventually be able to use the OpenID Connect (OIDC) flow for securing web browser access.

Reply:
I will now that I have a better idea how I'll use this. See this comment chain:
https://github.com/Sage-Bionetworks-Workflows/eks-stack/pull/35#discussion_r1797202156